source/fuzz/fuzzer.h

```diff
@@ -70,73 +75,64 @@ class Fuzzer {
 
   ~Fuzzer();
 
-  // Sets the message consumer to the given |consumer|. The |consumer| will be
-  // invoked once for each message communicated from the library.
-  void SetMessageConsumer(MessageConsumer consumer);
-
-  // Transforms |binary_in| to |binary_out| by running a number of randomized
-  // fuzzer passes.  Initial facts about the input binary and the context in
-  // which it will execute are provided via |initial_facts|.  A source of donor
-  // modules to be used by transformations is provided via |donor_suppliers|.
-  // The transformation sequence that was applied is returned via
-  // |transformation_sequence_out|.
-  FuzzerResultStatus Run(
-      const std::vector<uint32_t>& binary_in,
-      const protobufs::FactSequence& initial_facts,
-      const std::vector<fuzzerutil::ModuleSupplier>& donor_suppliers,
-      std::vector<uint32_t>* binary_out,
-      protobufs::TransformationSequence* transformation_sequence_out);
+  // Transforms |binary_in_| by running a number of randomized fuzzer passes.
+  // Initial facts about the input binary and the context in which it will
+  // execute are provided via |initial_facts_|.  A source of donor modules to be
+  // used by transformations is provided via |donor_suppliers_|.  On success,
+  // returns the transformed binary and the sequence of transformations that
+  // were applied.
+  FuzzerResult Run();
 
```

Comment:
Maybe it's better to explicitly mention here that `FuzzerResult::trasformed_binary` and `FuzzerResult::applied_transformations` are empty if `FuzzerResult::status` is not `kComplete`. @afd, what do you think?

If you agree, don't forget to adjust your comments in shrinker and replayer as well.

Reply:
Done.